op_crates/webgpu/01_webgpu.js

```diff
@@ -5012,8 +5012,16 @@
   }
   GPUObjectBaseMixin("GPUQuerySet", GPUQuerySet);
 
+  let instance;
+
   window.__bootstrap.webgpu = {
-    gpu: webidl.createBranded(GPU),
+    get gpu() {
```

Comment:
I think this `get` might run at the wrong time... we "get" the `gpu` when assigning it to navigator, right? Have you tested this works?

Reply:
this works, but thats because we `get` the navigator, so a little bit hacky. i can change it to something more appropriate